scripts/upgradeToV2/upgradeToV2.ts

```diff
@@ -320,6 +322,18 @@ export const main = async (networkName = name) => {
 
     release["StakingContracts"] = StakingContracts;
     release["DonationsStaking"] = DonationsStaking;
+    release["Identity"] = dao.Identity;
+    release["GoodDollar"] = dao.GoodDollar;
+    release["Contribution"] = dao.Contribution;
+    release["ForeignBridge"] = dao.ForeignBridge;
+    release["HomeBridge"] = dao.HomeBridge;
+    release["Controller"] = dao.Controller;
+    release["Avatar"] = avatar;
+    release["FirstClaimPool"] = dao.FirstClaimPool;
+    release["Reputation"] = dao.Reputation;
+    release["AbsoluteVote"] = dao.AbsoluteVote;
+    release["SchemeRegistrar"] = dao.SchemeRegistrar;
+    release["UBI"] = dao.UBI;
```

Comment:
add only contracts we DONT UPGRADE.
still not all non upgraded contracts are added
remove undefined such as homebridge on mainnet

Reply:
Removed them